e2e/cypress/integration/files_and_attachments/file_previews_5_spec.js

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @files_and_attachments
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+import {downloadAttachmentAndVerifyItsProperties} from './helpers';
+
+describe('Upload Files - Failing cases', () => {
+    let testTeam;
+
+    beforeEach(() => {
+        // # Login as sysadmin
+        cy.apiAdminLogin();
+
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+            cy.get('#post_textbox', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible');
+        });
+    });
+
+    it('MM-T2264_3 - BMP', () => {
+        const properties = {
+            route: 'mm_file_testing/Images/BMP.bmp',
+            originalWidth: 400,
+            originalHeight: 479,
+        };
+
+        testImage(properties);
+    });
+
+    it('MM-T2264_6 - PSD', () => {
+        const properties = {
+            route: 'mm_file_testing/Images/PSD.psd',
+            originalWidth: 400,
+            originalHeight: 479,
+        };
+
+        testImage(properties);
+    });
+
+    it('MM-T2264_15 - MOV', () => {
+        const properties = {
+            route: 'mm_file_testing/Video/MOV.mov',
+            shouldPreview: true,
+        };
+        testVideoFile(properties);
+    });
+
+    it('MM-T2264_20 - M4A', () => {
+        const properties = {
+            route: 'mm_file_testing/Audio/M4A.m4a',
+            shouldPreview: true,
+        };
+        testAudioFile(properties);
+    });
+
+    it('MM-T2264_22 - FLAC', () => {
+        const properties = {
+            route: 'mm_file_testing/Audio/FLAC.flac',
+            shouldPreview: true,
+        };
+        testAudioFile(properties);
+    });
+});
+
+function testAudioFile(fileProperties) {
```

Comment:
Some of the helper functions were repeated. Can those be extracted into a separate helper file within the same folder?

Reply:
Sure

Reply:
@saturninoabril done.